### PR TITLE
Nn 1969 remove update button

### PR DESCRIPTION
**Pull request by JamieDraperUK**

### CODE REVIEW

File: src/ResultsHouseblock/ResultsHouseblockContainer.js [hunk 1]
```diff
@@ -48,6 +49,26 @@ class ResultsHouseblockContainer extends Component {
     }
   }
 
+  async componentDidUpdate(prevProps) {
```
Comment (jsrobertson) [inline src/ResultsHouseblock/ResultsHouseblockContainer.js]: Is this any better?
```
if (
  (prevProps.date && prevProps.date !== date) ||
  (prevProps.period && prevProps.period !== period) ||
  (prevProps.currentLocation && prevProps.currentLocation !== currentLocation) ||
  (prevProps.currentSubLocation && prevProps.currentSubLocation !== currentSubLocation)
) {
      await this.update()
}
```

Comment (jsrobertson) [reply src/ResultsHouseblock/ResultsHouseblockContainer.js]: Same in `/ResultsActivityContainer.js` too.

File: src/App.scss [hunk 2]
```diff
@@ -225,6 +225,10 @@ table tr:nth-child(even){
   margin-bottom: 30px;
 }
 
+.pull-right {
```
Comment (jsrobertson) [inline src/App.scss]: 😢

File: src/Flags/flags.scss [hunk 3]
```diff
@@ -1,5 +1,9 @@
 .results-houseblock, .results-activity {
 
+  hr {
```
Comment (jsrobertson) [inline src/Flags/flags.scss]: 😢

File: src/App.scss (hunk 2 above)
Comment (JamieDraperUK) [reply src/App.scss]: I know, the whole page needs sorting out.

**Pull request merged**